Dockerfile

```diff
@@ -67,7 +67,7 @@ ADD aws-sudo/aws-sudo.sh /usr/local/bin/aws-sudo.sh
 
 # google-sdk
 RUN DEBIAN_FRONTEND=noninteractive apt-get -q install -y google-cloud-sdk \
-                                                         kubectl && \
+    kubectl && \
```

Comment:
Please leave the continuation indents in the apt lines

Reply:
Yes, Sir.